pyproject.toml

```diff
@@ -101,6 +101,15 @@ beautifulsoup4 = "^4.13.3"
 vprof = "^0.38"
 tuna = "^0.5.11"
 python-dotenv = "^1.1.0"
+sphinx = [
+  { version = "^7.4.7", python = ">=3.9,<3.11" },
+  { version = "^8.2.3", python = ">=3.11" }
+]
+myst-parser = [
+  { version = "^3.0.1", python = ">=3.9,<3.10" },
+  { version = "^4.0.1", python = ">=3.10" }
+]
+
```

Comment:
A bit unrelated to this admin change, I figured out how to get the sphinx dev dependencies working even with our project supporting back to 3.9 by specifying different deps for different python versions. This was helpful for iterating on docs and making sure things look good in the rendered output.